AzureCalling/Views/ParticipantView.swift

```diff
@@ -105,9 +108,44 @@ class ParticipantView: UIView {
         }
 
         placeholderImage.isHidden = isDisplayVideo
+        switchCameraButton.isHidden = !isDisplayVideo
         view.isHidden = !isDisplayVideo
     }
 
+    func updateCameraSwitch(isOneOnOne: Bool) {
+        // Skip update if display mode is the same
+        guard detachMode != isOneOnOne else {
+            return
+        }
+        detachMode = isOneOnOne
+
+        let buttonLength: CGFloat
+        let verticalConstraint: NSLayoutConstraint
+
+        if isOneOnOne {
+            // Top right (smaller) camera switch for detached local participant view
+            buttonLength = 24
+            verticalConstraint = switchCameraButton.topAnchor.constraint(equalTo: containerView.topAnchor, constant: 4)
+        } else {
+            // Middle right camera switch for standard local participant view
+            buttonLength = 30
+            verticalConstraint = switchCameraButton.centerYAnchor.constraint(equalTo: containerView.centerYAnchor)
+        }
```

Comment:
The middle right camera is to prevent the camera switch button to be covered by the InfoHeader when the caller is the only user in the call and have camera turned on.

When the call grid is is in PIP the switchCameraButton is constraint to the top-right corner. When call view grid is 2x2 or 3x2, the switchCamera Button should also be on the top-right corner. Probably can do a check if callingContext.displayedRemoteParticipants is not empty.

Reply:
I've thought about this as well, but decided to keep the button on middle right to be consistent on all views except for PIP. It depends on how we want the design to be.

@Gemakk how did you implement this in Android? Do you think we should keep it top-right corner for 2x2 and 3x2 as well?

Reply:
Default is right.  Top right only during 1:1